src/simulation/openables/BrimstoneChest.ts

```diff
@@ -0,0 +1,41 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+const BrimstoneChestTable = new LootTable()
+	.add('Uncut diamond', [25, 35], 1000)
+	.add('Uncut ruby', [25, 35], 1000)
+	.add('Coal', [300, 500], 1000)
+	.add('Coins', [50000, 150000], 1000)
+	.add('Gold ore', [100, 200], 800)
+	.add('Dragon arrowtips', [50, 200], 800)
+	.add('Iron ore', [350, 500], 600)
+	.add('Rune full helm', [2, 4], 600)
+	.add('Rune platebody', [1, 2], 600)
+	.add('Rune platelegs', [1, 2], 600)
+	.add('Raw manta ray', [80, 160], 600)
```

Comment:
the fish drops are dynamic based on the players fishing level.

can you try to make a new class called BrimestoneChest that extends SimpleOpenable and overload its open function to accept a fishing level to set the table that way